### PR TITLE
Minor clarifications in cluster install pre-reqs

### PR DESCRIPTION
**Pull request by charlieegan3**

### CODE REVIEW

File: docs/installation_manual_in_cluster.md [hunk 1]
```diff
@@ -76,7 +77,7 @@ If you built your own Docker image for Preflight, you need to edit `deployment/k
 We also need to customize some things in the configuration file. Edit `deployment/kubernetes/overlays/scanner/config/preflight.yaml` and chagne:
```
Comment (wwwil) [inline docs/installation_manual_in_cluster.md]:
```suggestion
We also need to customize some things in the configuration file. Edit `deployment/kubernetes/overlays/scanner/config/preflight.yaml` and change:
```
Just noticed this typo, not related to any change in this PR but would be a place time to fix it.

File: deployment/kubernetes/overlays/scanner/config/preflight.yaml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # this cluster-name is just used to mark the generated reports
 cluster-name: mycluster
+bucket-name: mybucket
```
Comment (wwwil) [inline deployment/kubernetes/overlays/scanner/config/preflight.yaml]: I don't think you can specify a bucket here.

Comment (charlieegan3) [reply deployment/kubernetes/overlays/scanner/config/preflight.yaml]: ooops, that is mistake.

**Comment by wwwil** [comment]
This PR will close #22

**Pull request merged**